examples/tomography.py

```diff
@@ -21,7 +21,7 @@
 """
 
 # Imports for common Python 2/3 codebase
-from __future__ import print_function, division
+from __future__ import print_function, division, absolute_import
```

Comment:
The import prevents implicit relative imports, see [here](http://python-future.org/compatible_idioms.html#imports-relative-to-a-package). It can (and should IMO) safely be added in any module.